src/sentry/static/sentry/app/components/events/interfaces/breadcrumbsV2/breadcrumbs.tsx

```diff
@@ -193,9 +196,14 @@ class Breadcrumbs extends React.Component<Props, State> {
     return match[1];
   };
 
-  getVirtualCrumb = (): Breadcrumb | undefined => {
+  getVirtualCrumb = (breadcrumb: Breadcrumb): Breadcrumb | undefined => {
     const {event} = this.props;
 
+    const timestamp = `${breadcrumb?.timestamp?.slice(
+      0,
+      ISO_STRING_DATE_AND_TIME_DIVISION
+    )}${event.dateCreated?.slice(ISO_STRING_DATE_AND_TIME_DIVISION)}`;
```

Comment:
What happens here when breadcrumb.timestamp is missing? Is an invalid timestamp created?